muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/dropdown.scss

```diff
@@ -143,3 +143,9 @@
     position: relative;
   }
 }
+
+.dropdown--communicator-labels {
+  .dropdown__container {
+    overflow-y: auto;
```

Comment:
Do we need max-height here or is it present somewhere else?

Reply:
By default dropdown__container has `max-height` property set to 350px